### ISSUE TITLE
Include commit links for maintainer:release-notes

### ISSUE THREAD

**Issue by colans** [mglaman/drupalorg-cli#37]
It would be nice if commit links could be added to `mrn` just like [Git Release Notes for Drush](https://www.drupal.org/project/grn) does.

**Comment by greggles** [comment]
Which commit links are these? It links to the issue, which links to the commits, so at least there's a workaround.

**Comment by mglaman** [comment]
🤔This is a feature request for `--commit-links`, correct?

````
```php
    // If the command was invoked with the --commit-links option
    if (!empty($commit_path)) {
      $item .= ' (<a href="' . $commit_path . $hash . '" title="View commit">#</a>)';
    }
```
```

It would append `(HASH)` after the issue. @colans is this the request?

**Comment by colans** [comment]
@mglaman   Yes, exactly.  It should be noted that these URLs changed recently now that the code is on GitLab.  (Because of this, and earlier changes to d.o's infrastructure, this is actually broken in `grn` as well.  See [Commit links are incorrect](https://www.drupal.org/project/grn/issues/2960856) for details.)

**Comment by mglaman** [comment]
Thanks for confirming and linking to the relevant issue! This shouldn't be _too_ hard to implement.
